packages/core/src/server/index.tsx

```diff
@@ -79,7 +79,7 @@ export default ({ packages }): ReturnType<Koa["callback"]> => {
     // Run beforeSSR actions.
     await Promise.all(
       Object.values(store.actions).map(({ beforeSSR }) => {
-        if (beforeSSR) return beforeSSR();
+        if (beforeSSR) return beforeSSR(ctx);
```

Comment:
I would make this an object, just in case we need to add more stuff in the future:
```js
beforeSSR({ ctx });
```